src/CpuCoreCounter.php

```diff
@@ -36,6 +36,19 @@ public function __construct(?array $finders = null)
         $this->finders = $finders ?? FinderRegistry::getDefaultLogicalFinders();
     }
 
+    /**
+     * @param positive-int $reservedCpus
+     *
+     * @return positive-int
+     */
+    public function getAvailableParallelism(int $reservedCpus = 1): int
```

Comment:
```suggestion
    public function getAvailableForParallelisation(int $reservedCpus = 1): int
```
IMHO that would describe the intent better and is grammatically correct.

Reply:
PS. Parallelism vs Parallelisation is subjective, I mostly miss "For" there to read it as a query sentence.